example/main.tf

```diff
@@ -66,11 +66,13 @@ resource "aws_key_pair" "example" {
 }
 
 module "ami" {
-  source = "github.com/Gabriella439/terraform-nixos-ng//ami"
+  source = "../ami"
```

Comment:
Would it be possible to leave this as is?  I wanted the example template to match what they would write in their own projects